### DISCUSSION TITLE
RuntimeError...;  Error:Incorrect args for pair coefficients (src/input.cpp:1713); Last command:pari_coeff

### DISCUSSION THREAD

**Discussion by liudfgoo** [deepmodeling/dpgen#588] [Q&A] [answered]
Hello, everyone!
An error occurred in the "model_devi" stage when running the dpgen. And the contents of "**_record.dpgen_**" file are as follows:
![image](https://user-images.githubusercontent.com/49781630/140845822-123d460f-f9f3-40b0-b09d-77d4a093ec4c.png)
The figures below are  "_**input.lammps**_" and "**_log.lammps_**" respectively.
![image](https://user-images.githubusercontent.com/49781630/140846523-e504db53-a993-47f1-b929-647aeefa11c3.png)

![image](https://user-images.githubusercontent.com/49781630/140845563-a1e76af7-1781-4826-b555-3949dc614e8a.png)
The error says "**Incorrect args for pair coefficients...**".  But I think there is no problem in the  "pair_style" and "pair_coeff" command settings.  And now I don't know how to solve this problem. Looking forward to your reply!
Thanks in advance!

**Comment by njzjz** [comment, marked as answer]
See #567; use `pair_coeff * *` instead

**Reply by liudfgoo** [reply]
Oh! I see. Thanks !
